tests/phpunit/integration/bootstrap.php

```diff
@@ -7,46 +7,64 @@
  * @link        http://www.getbeans.io
  * @license     GNU-2.0+
  *
- * @group       integrationtests
+ * @group       integration-tests
  */
 
 if ( ! file_exists( '../../../wp-content' ) ) {
-	return;
+	trigger_error( 'Unable to run the integration tests, as the wp-content folder does not exist.', E_USER_ERROR );  // @codingStandardsIgnoreLine.
 }
 
+define( 'BEANS_TESTS_DIR', __DIR__ );
 define( 'BEANS_THEME_DIR', dirname( dirname( dirname( __DIR__ ) ) ) );
 define( 'WP_CONTENT_DIR', dirname( dirname( dirname( getcwd() ) ) ) . '/wp-content/' ); // @codingStandardsIgnoreLine.
 
 if ( defined( 'WP_CONTENT_DIR' ) && ! defined( 'WP_PLUGIN_DIR' ) ) {
 	define( 'WP_PLUGIN_DIR', WP_CONTENT_DIR . 'plugins/' ); // @codingStandardsIgnoreLine.
 }
 
-$beans_tests_dir = getenv( 'WP_TESTS_DIR' );
+/**
+ * Get the WordPress' tests suite directory.
+ *
+ * @since 1.5.0
+ *
+ * @return string
+ */
+function beans_get_wp_tests_dir() {
+	$tests_dir = getenv( 'WP_TESTS_DIR' );
 
-// Travis CI & Vagrant SSH tests directory.
-if ( empty( $beans_tests_dir ) ) {
-	$beans_tests_dir = '/tmp/wordpress-tests';
-}
+	// Travis CI & Vagrant SSH tests directory.
+	if ( empty( $tests_dir ) ) {
+		$tests_dir = '/tmp/wordpress-tests';
+	}
 
-// Relative path to Core tests directory.
-if ( ! file_exists( $beans_tests_dir . '/includes/' ) ) {
-	$beans_tests_dir = '../../../../tests/phpunit';
-}
+	// If the tests' includes directory does not exist, try a relative path to Core tests directory.
+	if ( ! file_exists( $tests_dir . '/includes/' ) ) {
+		$tests_dir = '../../../../tests/phpunit';
+	}
 
-if ( ! file_exists( $beans_tests_dir . '/includes/' ) ) {
-	trigger_error( 'Unable to locate wordpress-tests-lib', E_USER_ERROR ); // @codingStandardsIgnoreLine.
+	// Check it again. If it doesn't exist, stop here and post a message as to why we stopped.
+	if ( ! file_exists( $tests_dir . '/includes/' ) ) {
+		trigger_error( 'Unable to run the integration tests, as the WordPress test suite could not be located.', E_USER_ERROR );  // @codingStandardsIgnoreLine.
```

Comment:
Just as a note, this would be a good place to refer to the wiki for test setup instructions. :)